helpers/kernel_version.go

```diff
@@ -0,0 +1,46 @@
+package helpers
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"syscall"
+)
+
+// UnameRelease gets the version string of the current running kernel
+func UnameRelease() string {
+	var uname syscall.Utsname
+	if err := syscall.Uname(&uname); err != nil {
+		return ""
+	}
+	var buf [65]byte
+	for i, b := range uname.Release {
+		buf[i] = byte(b)
+	}
+	ver := string(buf[:])
+	ver = strings.Trim(ver, "\x00")
+	return ver
+}
+
+func kernelIsAtLeast(verMajor, verMinor int) (bool, error) {
```

Comment:
Im merging today a change that will be more complete (generic usage) than this one (simple just for your needs). Is it possible for you to wait to use the OSInfo API ?

Reply:
![image](https://user-images.githubusercontent.com/7395852/131683631-39bd4468-852a-4e43-a62c-27b120257751.png)

Reply:
You will be able to run helpers.NewOSInfoRelease() to get the kernel version only (without /etc/os-release) and use its internal functions (comparing kernel versions etc).

Reply:
```
$ sudo ./dist/tracee-ebpf --debug --trace uid=1000 --trace pid=new --trace event=execve --trace event=cap_capable --output option:detect-syscall
OSInfo: PRETTY_NAME: "Ubuntu 21.04"
OSInfo: VERSION_ID: "21.04"
OSInfo: VERSION_CODENAME: hirsute
OSInfo: KERNEL_RELEASE: 5.11.0-31-generic
OSInfo: VERSION: "21.04 (Hirsute Hippo)"
OSInfo: ID: ubuntu
OSInfo: ID_LIKE: debian
TEST: ubuntu
TEST: 5.11.0-31-generic
BTF: bpfenv = false, btfenv = false, vmlinux = true
BPF: using embedded BPF object
unpacked CO:RE bpf object file into memory
```

Reply:
Could you please review https://github.com/aquasecurity/libbpfgo/pull/67 and see if that would satisfy your needs on this one ?

Reply:
You may use either:

`OSInfo.CompareOSBaseKernelRelease("5.11.0"))`

or just the function

`CompareOSBaseKernelRelease(baseVersion, givenVersion)`

from the API.

Reply:
yes it looks very good and should address all my use cases, thank you for that